extensions/amp-sidebar/1.0/example/index.html

```diff
@@ -1,53 +1,111 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <script
-      type="module"
-      async
-      src="https://cdn.ampproject.org/bento.mjs"
-    ></script>
-    <script nomodule src="https://cdn.ampproject.org/bento.js"></script>
-    <script
-      type="module"
-      async
-      src="https://cdn.ampproject.org/v0/bento-sidebar-1.0.mjs"
-    ></script>
-    <script
-      nomodule
-      async
-      src="https://cdn.ampproject.org/v0/bento-sidebar-1.0.js"
-    ></script>
-    <link
-      rel="stylesheet"
-      type="text/css"
-      href="https://cdn.ampproject.org/v0/bento-sidebar-1.0.css"
-    />
-  </head>
-  <body>
-    <bento-sidebar id="sidebar1" side="right">
-      <ul>
-        <li>Nav item 1</li>
-        <li>Nav item 2</li>
-        <li>Nav item 3</li>
-        <li>Nav item 4</li>
-        <li>Nav item 5</li>
-        <li>Nav item 6</li>
-      </ul>
-    </bento-sidebar>
-
-    <div class="buttons" style="margin-top: 8px">
-      <button id="open-sidebar">Open sidebar</button>
-    </div>
-
-    <script>
-      (async () => {
-        const sidebar = document.querySelector('#sidebar1');
-        await customElements.whenDefined('bento-sidebar');
-        const api = await sidebar.getApi();
-
-        // set up button actions
-        document.querySelector('#open-sidebar').onclick = () => api.open();
-      })();
-    </script>
-  </body>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>bento-sidebar</title>
+
+    <head>
+      <script src="https://cdn.ampproject.org/bento.js"></script>
+      <link
+          rel="stylesheet"
+          type="text/css"
+          href="https://cdn.ampproject.org/v0/bento-sidebar-1.0.css"
+          />
+      <script
+          async
+          src="https://cdn.ampproject.org/v0/bento-sidebar-1.0.js"
+          ></script>
+      <style>
+body {
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto,
+  Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji',
+  'Segoe UI Symbol';
+background: #ecf1f3;
+}
+
+    button {
+      -webkit-appearance: none;
+      background: transparent;
+      border: none;
+      box-shadow: none;
+      color: #4c5a63;
+      cursor: pointer;
+      float: right;
+      width: 40px;
+      height: 35px;
+      margin: 0 1rem;
+    }
+
+    h2 {
+      font-family: 'Inter', sans-serif;
```

Comment:
Inter doesn't appear defined in this stylesheet or elsewhere on the document.

Reply:
removed